### ISSUE TITLE
Crash at City-State turn (during turn transition 219 to 220)

### ISSUE THREAD

**Issue by Maifailun** [LoneGazebo/Community-Patch-DLL#3176]
_1. Mod version (i.e Date - 4/23):_

Latest version that can be downloaded at DLC Modpack and 43-Civ Version thread. https://forums.civfanatics.com/threads/dlc-modpack-and-43-civ-version-repository.609278/
.

_2. Mod list (if using Vox Populi only, leave blank):_

24.03.2017  02:12    <DIR>          (1) Community Patch
24.03.2017  02:12    <DIR>          (2) Community Balance Overhaul
24.03.2017  02:12    <DIR>          (3) CSD for CBP
24.03.2017  02:12    <DIR>          (4) Civ IV Diplomatic Features
24.03.2017  02:12    <DIR>          (5) More Luxuries
24.03.2017  02:12    <DIR>          (6c) 43 Civs CP
16.03.2017  00:26    <DIR>          (7) Community Events (v 3)
15.03.2017  23:42    <DIR>          (8) Events and Decisions (CBO) (v 13)
27.03.2017  02:02    <DIR>          46926-historical_religions_complete__bnw_or_gk___v_45
30.03.2017  16:12    <DIR>          City State Selection (v 1)
15.03.2017  22:48    <DIR>          Advanced Unique Units (v 1)
15.03.2017  22:48    <DIR>          Architecture changes (v 1)
26.03.2017  17:35    <DIR>          Assyrian Lamassu (v 1)
15.03.2017  23:01    <DIR>          Builders Heaven (v 1)
16.03.2017  00:33    <DIR>          Buildings - Goldsmith (v 4)
16.03.2017  00:33    <DIR>          Buildings - Guilds Great Work Displays (v 2)
16.03.2017  00:33    <DIR>          Buildings - Lighthouse Near Sea (v 2)
16.03.2017  00:33    <DIR>          Buildings - Textile Mill (v 4)
16.03.2017  00:45    <DIR>          Cities of Marble (BNW) (v 6)
15.03.2017  23:26    <DIR>          Civ Names by Policies (v 8)
15.03.2017  22:52    <DIR>          Community Patch Mod Compatibility (v 14)
02.04.2017  17:10    <DIR>          Culturally Linked Bias~
26.03.2017  17:35    <DIR>          Egyptian Nilometer (v 1)
15.03.2017  23:20    <DIR>          Emigration (v 6)
15.03.2017  23:01    <DIR>          Faster Aircraft Anim~
15.03.2017  23:01    <DIR>          Faster Plot
15.03.2017  22:48    <DIR>          Global - 2 Units Per Tile (v 2)
15.03.2017  23:26    <DIR>          Global - Counter Espionage (v 3)
15.03.2017  23:26    <DIR>          Global - Espionage Mapping (v 7)
16.03.2017  15:25    <DIR>          Global - Espionage Race (v 3)
15.03.2017  22:48    <DIR>          Global - GA Happiness (v 1)
15.03.2017  22:48    <DIR>          Global - Great Work Vaults (v 2)
15.03.2017  22:48    <DIR>          Global - Pro-rata Buildings Purchase (v 2)
15.03.2017  22:48    <DIR>          Global - Religious Settlers (v 1)
29.03.2017  16:24    <DIR>          Greek Agora (v 1)
16.03.2017  00:37    <DIR>          Improved Air Recon (v 1)
02.04.2017  17:10    <DIR>          InfoAddict (v 22)
30.01.2015  06:57    <DIR>          Mechanized Infantry with Soldiers, Resized (v 2)
15.03.2017  22:51    <DIR>          More War (v 3)
15.03.2017  22:51    <DIR>          movementmod
16.03.2017  10:07    <DIR>          No More Tech Tree Beelining (v 4)
15.03.2017  22:51    <DIR>          Ocean Double Move (v 1)
15.03.2017  22:51    <DIR>          Promotions - Expansion Pack (v 9)
15.03.2017  22:51    <DIR>          Promotions - No InstaHeal (v 3)
15.03.2017  22:51    <DIR>          Quick Turns (v 10)
15.03.2017  22:52    <DIR>          R.E.D. Modpack (v 27)
16.03.2017  00:31    <DIR>          Really Advanced Setup (v 15)
16.03.2017  00:31    <DIR>          Reforestation (v 8)
15.03.2017  22:52    <DIR>          Slower Science Longer Eras (v 1)
15.03.2017  22:52    <DIR>          Titans of Commerce Science and Industry (BNW) (v 5)
15.03.2017  23:30    <DIR>          UI - Destination (v 4)
15.03.2017  22:51    <DIR>          UI - Overlay Antiquities (v 2)
15.03.2017  22:51    <DIR>          UI - Overlay Barbarians (v 8)
15.03.2017  22:51    <DIR>          UI for Unit Stacking (v 1)
23.03.2017  00:05    <DIR>          Units - Multiple Upgrades (v 2)
15.03.2017  22:51    <DIR>          upgrade building~
16.03.2017  00:38    <DIR>          useful oil mod (v 10)
15.03.2017  22:51    <DIR>          War - Less Damaged Captured Cities (v 3)
02.04.2017  17:00    <DIR>          XP From Forts (v 1)

_3. Error description:_ 

Crashes on turn 220, when its City States' turn. Attached you can find turn 219 quicksave along with all previous saves/games with same cause of crashes.
Crashes has been happening no matter what I tried in order to play the game along with my previous mods. After 8 trials of new games (at marathon speed I might add) I am confident it always crashes at city state phase. 

Most of installed mods seem to be minor/non conflicting. I should mention that I paid attention to compatible mods thread while using those alongside with CP+CBP.

Some of these mods were unnecessary (such as Builders Heaven, it was just a left over, I figured CBP overwrites it, I kept it anyway just to load my save without issues related to it.)

**Thank you in advance.**

_4. Steps to reproduce (optional):_

---------------------------
**I couldn't upload zip file for some reason so I post here a dropbox link to the related [zip files](
https://www.dropbox.com/s/yjzaj92n70gow7u/Crash%20Files%2002042017.zip?dl=0).**

Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by Maifailun** [comment]
Hi,

So I played a new game with similar setup but I deactivated the mods XP from Forts, Cultural Linked Bias and Info Addict. Now I crashed again but at turn 210, while it was processing the turn for City States.

I checked logs (net_message_debug file) and it mentions player 43 (city state mombasa) last and stops there. [Here is](https://www.dropbox.com/s/1amuw1zldbhblea/New%20Crash%20Files%2001042017.zip?dl=0) new logs/minidump for the crash debugging.
